homeassistant/components/apcupsd/sensor.py

```diff
@@ -461,6 +462,16 @@ async def async_setup_platform(
         "from your configuration.yaml file"
     )
 
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecated_yaml",
+        breaks_in_ha_version="2022.10.0",
```

Comment:
Our normal deprecation period is two releases.
```suggestion
        breaks_in_ha_version="2022.12.0",
```